src/pages/meetingsPage.jsx

```diff
@@ -1,132 +1,191 @@
-              WINTER
-            </button>
-            <button className={`tab font-inter font-medium text-black ${activeTab === 'SPRING' ? 'tab-active' : ''}`} style={{letterSpacing: '2px'}} onClick={() => setActiveTab('SPRING')}>
-              SPRING
-            </button>
+  function RenderMeetings({MeetingList}){
+  
+    return(
+      <div className="relative overflow-hidden px-72 space-y-24 my-52 bg-white">
+        <img src={HalfGear} className="absolute right-0"/>
+        <div className="flex flex-row justify-between items-center">
+          <div>
+            <p className="font-[#7D7D7D] text-xl font-semibold">
+              &gt;&gt; ‘24 -‘25
+            </p>
+            <p className="text-5xl font-semibold">
+              {activeQuarter} Quarter Meetings
+            </p>
           </div>
-          
-          <div className="tab-content p-10 text-black">
-            {tabContent[activeTab].map((section, index) => (
-              <InfoSection
-                key={index}
-                title={section.title}
-                date={section.date}
-                pillar={section.pillar}
-                description={section.description}
-              />
-            ))}
+          <div>
+            <QuarterButton Quarter="Fall" />
+            <QuarterButton Quarter="Winter" />
+            <QuarterButton Quarter="Spring" />
           </div>
-
-          
         </div>
-      </div>
-
-      <div style={{ position: 'relative', width: '100%', height: '100vh', overflow: 'hidden', marginTop: '-600px' }}>
-        <div className="absolute z-0" style={{ top: '-190px', left: '-10px', width: '600px' }}>
-          <img src={Sun} alt="Sun" style={{ width: '100%', height: 'auto'}} className="object-cover" />
+        <ul className="space-y-28">
+          {MeetingList[activeQuarter].map((meeting) => (<Meeting title={meeting.title} pillar={meeting.pillar} date={meeting.date} description={meeting.description} image={meeting.image}/>))}
+        </ul>
+        <div className="flex flex-row justify-between items-center">
+            <button className="bg-black text-white font-inter text-xl font-semibold leading-[35px] tracking-widest text-left px-4 py-2 flex items-center justify-between">
+              <span>VIEW MORE</span>
+              <img src={ViewMoreButtonVector} alt="> arrow" className="ml-2 w-3 h-5" />
+            </button>
+            <div>
+              <QuarterButton Quarter="Fall" />
+              <QuarterButton Quarter="Winter" />
+              <QuarterButton Quarter="Spring" />
+            </div>
         </div>
       </div>
+    )
+  }
+  return (
+    <div>
+      <Navbar title="MEETINGS" subtitle="WEDNESDAYS 7-9 PM" bg={MeetingsBG}/>
+      <div className="flex flex-col items-center justify-center w-full min-h-screen bg-white">
```

Comment:
for the min-h-screen on this one, it makes this blurb section really long when we really want it be much smaller than that, I would look at the figma for that